src/authentication/oauth2Auth.ts

```diff
@@ -714,7 +714,7 @@ export class Oauth2Auth extends AlfrescoApiClient {
     }
 
     exchangeTicketListener(alfrescoApi: AlfrescoApi) {
-        this.once('token_issued', async () => {
+        this.on('token_issued', async () => {
```

Comment:
this needs unit tests to avoid problems in the future

Reply:
I think it make sense, the only problem I see is that it will refresh the half token on the same lifespan of the SSO token. that is ok If the system is kind of correctly configured but could happen that the same system has a different lifespan for thoese

Reply:
AFAIK we are not  creating new session for each sso refresh
Here what ever we do that doesn't create any session rather it increments the current lifespan the session.

Reply:
I can confirm the above behaviour i tried to keep the stale oauth session for 2 hours . i didn't see any new session created in the acs and i can still access viewer without any problem